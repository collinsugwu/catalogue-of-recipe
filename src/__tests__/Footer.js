import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Footer from '../components/Footer';


afterEach(cleanup);
it('renders', () => {
  const { asFragment } = render(<Footer />);
  expect(asFragment()).toMatchSnapshot();
});

it('containers Enjoy in p tag', () => {
  const { getByText } = render(<Footer />);
  expect(getByText('Enjoy Â© 2020')).toBeVisible();
});
