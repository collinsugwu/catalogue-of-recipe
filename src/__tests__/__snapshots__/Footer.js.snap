// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <footer>
    <div
      class="info"
    >
      <p>
        Enjoy Â© 2020
      </p>
    </div>
  </footer>
</DocumentFragment>
`;
